src/libraries/System.IO.Packaging/src/System/IO/Packaging/ZipPackage.cs

```diff
@@ -393,7 +397,7 @@ internal static string GetOpcNameFromZipItemName(string zipItemName)
                     break;
                 case CompressionOption.SuperFast:
                     {
-                        compressionLevel = CompressionLevel.Fastest;
+                        compressionLevel = CompressionLevel.NoCompression;
```

Comment:
This would be a breaking change, but if it's more appropriate, we can take it.

@edwardneal @maedula is this the preferred/more correct `CompressionLevel` value when converting from XPS?